<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrivateClient</class>
 <widget class="QWidget" name="PrivateClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="pri_lbNick">
       <property name="text">
        <string>Nick</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pri_lbAddress">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pri_lbPort">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="pri_editSent">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_6">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="pri_editInput">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::WidgetWidth</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pri_btnClose">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pri_btnSendmsg">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pri_btnClose</sender>
   <signal>clicked()</signal>
   <receiver>PrivateClient</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
